tsconfig.json

```diff
@@ -2,17 +2,16 @@
   "compilerOptions": {
     "declaration": true,
     "outDir": "lib",
+    "sourceMap": true,
     "esModuleInterop": true,
     "module": "commonjs",
     "moduleResolution": "node",
-    "jsx": "react",
     "resolveJsonModule": true,
-    "target": "es6",
-    "allowSyntheticDefaultImports": true,
-    "allowJs": true,
-
+    "target": "es2017",
+    "skipLibCheck": true,
```

Comment:
We don't want `skipLibCheck` unless absolutely necessary - it can allow `any` to sneak into the types if something goes wrong in node_modules.